Classes/OGVPlayerState.m

```diff
@@ -111,33 +120,26 @@ -(void)seek:(float)time
     }
     dispatch_async(decodeQueue, ^() {
         if (decoder && decoder.seekable) {
-            BOOL wasPlaying = !self.paused;
-            if (wasPlaying) {
+            wasPlayingBeforeSeek = !self.paused;
```

Comment:
This seems to get confused when running multiple seeks, such as scrubbing in the example app. In the current code, seeking to a keypoint is a single atomic operation on the decodeQueue thread, but in this code, the seeking operation extends beyond that to further call-ins as we iterate through audio and video packets looking for our final target. As a result, when I scrub, the second seek sees that we're paused and sets wasPlayingBeforeSeek to false, so there's no continuation after seeking.